experimental/sev_guest/README.md

```diff
@@ -0,0 +1,5 @@
+# AMD SEV-SNP Guest Library
+
+This library implements Rust wrappers for new CPU instructions inroduced for AMD
+SEV, SEV-ES and SEV-SNP. It also adds structs that represent data structures
+used by AMD SEV-ES and SEV-SNP.
```

Comment:
nit: I'd add a link to https://www.amd.com/system/files/TechDocs/24594.pdf here. Or perhaps a link to AMD SEV-SNP white paper.

Reply:
Good idea. I added a number of links to reference docs.